[DOCS] Add Intro to Add an Expectation to a Data Asset Column (#9023)
great-expectations/great_expectations: docs/docusaurus/docs/cloud/data_assets/manage_data_assets.md

## Add an Expectation to a Data Asset column

> When you create an Expectation after fetching metrics for a Data Asset, the column names and some values are autopopulated for you and this can simplify the creation of new Expectations. Data Asset Metrics can also help you determine what Expectations might be useful and how they should be configured. When you create new Expectations after fetching Data Asset Metrics, you create the Expectation Suite and add Expectations to it at the same time. 

Reviewer: Thanks! This is great. Small suggestion to consider based off of the toggle functionality for add to new vs. existing suite:

```suggestion
When you create an Expectation after fetching metrics for a Data Asset, the column names and some values are autopopulated for you and this can simplify the creation of new Expectations. Data Asset Metrics can also help you determine what Expectations might be useful and how they should be configured. When you create new Expectations after fetching Data Asset Metrics, you can add them to an existing Expectation Suite, or you can create a new Expectation Suite and add the Expectations to it. 
```
